kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/repository/ConfigFileWatcher.java

```diff
@@ -55,10 +58,22 @@ public ConfigFileWatcher(String configFilePath, ConfigurationMarshaller marshall
             if(toWatch.toFile().exists()) {
                 lastUpdate = Files.getLastModifiedTime(toWatch).toMillis();
             } else {
+                String cfg = marshaller.marshall(configuration);
+                File file = new File(this.toWatch.toString() + "kie-server-router.json");
+                file.createNewFile();
+                if(file.exists()){
+                    try (FileOutputStream fos = new FileOutputStream(file); PrintWriter writer = new PrintWriter(fos)) {
+                        writer.write(cfg);
+                    }
+                }
+                FileTime lastModified = Files.getLastModifiedTime(toWatch);
+                lastUpdate =  lastModified.toMillis();
                 log.warnv("configuration file does not exist {0} ", this.toWatch);
```

Comment:
move this log line to th e beginning of the else clause and update it to something like, "configuration file does not exist {0}, creating..."